### PR TITLE
Allows other species to be hulk

### PR DESCRIPTION
**Pull request by blobbernaut**
Why is this even a feature? Somebody put this line here on purpose. Why.

### CODE REVIEW

**Comment by Crystalwarrior** [comment]
THANK YOU. No species in our codebase have any snowflakey "abilities", this had no reason to exist.

**Comment by Kierany9** [comment]
admin_ranks.txt was changed

what dastardly conspiracy are you plotting now?

**Comment by Crystalwarrior** [comment]
Holy fuck are you seriously trying to sneak in the moderator preferences
Are you fucking drunk?

**Comment by blobbernaut** [comment]
sorrry that was on a denied PR

**Comment by blobbernaut** [comment]
you remove that DNM right now mister

File: config/initial_config/admin_ranks.txt [hunk 1]
```diff
@@ -37,4 +37,4 @@ Game Master		= +EVERYTHING
 
 Host			= +EVERYTHING
 
-Coder			= +DEBUG +VAREDIT +SERVER +SPAWN
\ No newline at end of file
+Coder			= +DEBUG +VAREDIT +SERVER +SPAWN
```
Comment (Crystalwarrior) [inline config/initial_config/admin_ranks.txt:40]: discard changes to admin_ranks.txt on your branch

**Comment by TheOneAndOnlyCreeperJoe** [comment]
How about the species being visually distinguishable? Last time I checked you couldn't even see that most alien species had hulk.

**Pull request closed without merging**